content/documentation/commands.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "Commands"
+weight = 21
++++
+
+## build (Build the project)
+
+```bash
+vendor/bin/phel build
+```
+
+Build the current project into the `out-dir` folder. This means that the compiled phel code into PHP will be saved in that directory, so you can run the PHP code directly using the PHP interpreter. This will improve the runtime performance -because there won't be a need to compile the code again.
+
+[Configuration](/documentation/configuration/) in `phel-config.php`:
+```php
+<?php
+return [
+    // [...]
+    'out-dir' => 'out',
+];
+```
+
+## export (Export definitions)
+
+Export all definitions with the meta data `{:export true}` as PHP classes. 
+
+It generates PHP classes at namespace level and a method for each exported definition. This allows you to use the exported phel functions from your PHP code.
+
+```bash
+vendor/bin/phel export
+```
+
+[Configuration](/documentation/configuration/) in `phel-config.php`:
+```php
+<?php
+return [
+    // [...]
+    'export' => [
+        'directories' => ['src'],
+        'namespace-prefix' => 'PhelGenerated',
+        'target-directory' => 'src/PhelGenerated'
+    ]
+];
+```
+
+## format (Format phel files)
+
+Formats the given files. You can pass relative or absolute paths.
+
+```bash
+vendor/bin/phel format src tests
+```
+
+## repl (Read-eval-print loop)
```

Comment:
In my opinion, maybe this command should be placed at the top.

Reply:
I thought about this, but in the end I decided to leave it in an "alphanumerical order". But maybe we should sorter by other random criteria(?) 🤔 I am not sure about it. I will leave this decision to @jenshaase as well.